pages/checkout.php

```diff
@@ -1,689 +1,634 @@
-<?php

```

Comment:
changes the whole page because of the tab and/or spacing updates. We need a WPCS pass first so we can better merge this. I will not merge a PR that replaces the whole file. Too likely we miss something with so many updates going on.

Reply:
This may be fallout from merging with v2. I didn't replace the whole page when coding.